src/screens/user-dashboard.js

```diff
@@ -1,28 +1,40 @@
-import React from 'react';
-import { userSelector } from '../store/user/selectors';
-import { useSelector } from 'react-redux';
-import { history } from '../history';
-import { ROUTES } from '../constants';
+import React, { useEffect } from "react";
+import { userSelector } from "../store/user/selectors";
+import { toDoSelector } from "../store/to-do/selectors";
+import { useSelector } from "react-redux";
+import { history } from "../history";
+import { ROUTES } from "../constants";
+import { useDispatch } from "react-redux";
+import { to_do_list } from "../store/to-do/actions";
+import ToDoList from "../components/to-do-list";
 
 const UserDashboard = () => {
-    const user = useSelector(userSelector);
-    const logout = () => {
-        localStorage.clear();
-        history.push(ROUTES.LOGIN);
-    }
+  const user = useSelector(userSelector);
+  const todos = useSelector(toDoSelector);
+  const dispatch = useDispatch();
 
-    return (
+  const logout = () => {
```

Comment:
ovaj logout mozemo izvuci u neki Header komponentu pa u App.js to include-ovati
Ali aj nemoj to na ovom tasku